dockers/docker-orchagent/Dockerfile.j2

```diff
@@ -23,10 +23,13 @@ debs/{{ deb }}{{' '}}
 {%- endfor %}
 
 COPY start.sh /usr/bin/start.sh
+COPY config.sh /usr/bin/config.sh
+COPY ipinip.json.j2 /usr/share/sonic/templates/ipinip.json.j2
+COPY mirror.json.j2 /usr/share/sonic/templates/mirror.json.j2
 
 ## Clean up
 RUN apt-get clean -y; apt-get autoclean -y; apt-get autoremove -y
 RUN rm -rf /debs
 
-ENTRYPOINT ["/bin/bash"]
-CMD ["/usr/bin/start.sh"]
+ENTRYPOINT ["/bin/bash", "-c"]
```

Comment:
@marian-pritsak has a PR #441 that merges config.sh and start.sh back into a single file. I don't exactly know his reason, but I'll suggest you sync with him to see if he has any concern on using two scripts instead of one.